src/main/java/no/nav/veilarbarena/controller/AdminController.java

```diff
@@ -22,19 +24,22 @@
 
     private final AuthContextHolder authContextHolder;
 
-    private final KafkaRepubliseringService kafkaRepubliseringService;
-
     private final OppdaterteBrukereRepository oppdaterteBrukereRepository;
 
-    @PostMapping("/republiser/endring-pa-bruker")
-    public String republiserEndringPaBruker(@RequestParam(required = false, defaultValue = "0") int fromOffset) {
+    @PostMapping("/republiser/endring-pa-bruker/all")
+    public String republiserTilstand() {
         sjekkTilgangTilAdmin();
-        return JobRunner.runAsync("republiser-endring-pa-bruker", () -> kafkaRepubliseringService.republiserEndringPaBrukere(fromOffset));
+        return JobRunner.runAsync("legg-alle-brukere-pa-v2-topic",
+                () -> oppdaterteBrukereRepository.insertAlleBrukereFraOppfolgingsbrukerTabellen((Date.valueOf(LocalDate.now().plusYears(1))))
```

Comment:
Legge inn en kommentar, f.eks noe sånt: Bruker dato et år frem i tid for at løpende oppdateringer fra Arena skal få prioritet